docs/src/pages/styles/typography/responsive-typography.example.html

```diff
@@ -0,0 +1,16 @@
+<article class="ds-base--inverse ds-u-padding--2">
```

Comment:
This file doesn't need to be added - it's identical to the core

Reply:
Thank you, I'll remove it.